### DISCUSSION TITLE
Best way to work with Tailwind CSS ?

### DISCUSSION THREAD

**Discussion by jameswong3388** [ueberdosis/tiptap#2960] [General Questions]
Just started to work with TipTap recently, and was struggling to style the element within `editor` like `a`, `code`, `p` tag ... 

Any idea to work with Tailwind CSS ?  any answer would be appreciated with THX!

**Comment by svenadlung** [comment]
Good morning James, did you read https://tiptap.dev/guide/styling ? You can style with global css or custom element classes easily.

**Comment by rokde** [comment]
Your styling guide is wrong. You can not add the classes to `editorProps.attributes` - you have to use `editorProps.attributes.class` to assign classes.

```js
const editor = useEditor({
    editorProps: {
        attributes: {
            class: 'prose prose-sm sm:prose lg:prose-lg xl:prose-2xl m-5 focus:outline-none',
        },
    },
    // other settings
}
```
